src/dispensary.js

```diff
@@ -142,26 +142,30 @@ export default class Dispensary {
     var files = [];
 
     for (let version of library.versions) {
-      if (library.filename) {
-        files.push({
-          file: library.filename,
-          fileOut: library.filenameOutput || library.filename,
-          index: index,
-          library: library,
-          version: version,
-          minified: false,
-        });
-      }
+      var minVersion = library.minVersion;
+      if (!minVersion || (minVersion && naturalCompare(
+          minVersion, version) <= 0)) {
```

Comment:
Explaining why we want a result less than or equal to `0` from `naturalCompare` would help here. I don't know the API so I don't quite get what's going on.